src/client/components/page/page.image.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 <div class="lzyxtsnt">
-	<img v-if="image" :src="image.url"/>
+	<img v-if="image" :src="image.url" :alt="image.name"/>
```

Comment:
same reason

Reply:
@rinsuki 

Yes you are correct. Title or description is much better for alt. 

I used `image.name` because it was previously used in Misskey code
https://github.com/syuilo/misskey/blob/c6837b9fdfbc30d8e3ede381293281db2d6364f0/src/client/components/image-viewer.vue#L3

What is the correct way to display title or description for Misskey? Is it `image.title`?

Reply:
actually correct data of title (or description) is not available.
in database schema, we have `image.comment`, but we can't set any value from API, so this is dead property.

(but anyways we must provide alt-text from mastodon or other servers, so i will create issue about this)

Reply:
created: #6606